test/data_aggregator/collections/collection_test.exs

```diff
@@ -276,7 +275,11 @@ defmodule DataAggregator.CollectionTest do
 
       assert_lists_equal([], ValidationRequestRecordVersion.read!(tenant: collection))
 
-      assert_lists_equal([], Attachment.read!())
+      assert {:ok, attachments} = Attachment.read()
```

Comment:
If you filter out deleted from the primary read action, you can maybe test like this:

```elixir
deleted_attachments = Attachment.read_deleted!()
assert length(deleted_attachments) == 42
```